docs/msw/binaries.md

```diff
@@ -5,26 +5,23 @@ How to use wxMSW binaries              {#plat_msw_binaries}
 
 Supported Compilers
 -------------------
-We provide pre-built binary files for the following compilers:
+
+We provide pre-built binary files for x86-64 Windows for the following compilers:
 
 * Microsoft Visual C++ compiler versions 14.0, 14.1, 14.2 and 14.3
   (corresponding to marketing product names of Microsoft Visual Studio 2015, 2017, 2019 and 2022 respectively).
   Please note that MSVC 14.x versions are ABI-compatible and the same set of binaries is used for all of them.
-* MinGW-w64 versions 7.3 and 8.1 (32-bit binaries use SJLJ exceptions, 64-bit ones use SEH, and all binaries use Win32 threads).
-* [TDM-GCC](https://jmeubank.github.io/tdm-gcc/) 10.3.0.
-* [MSYS2](https://www.msys2.org/) MinGW 12.2.0 and 13.2.0.
+* [MSYS2](https://www.msys2.org/) MinGW 15.1.
```

Comment:
```suggestion
* [MSYS2](https://www.msys2.org/) environment MINGW64 (GCC 15.1).
```
Just to use the official MSYS2 environment name. MinGW there can also use clang, I think. I guess it was decided to stick to the old MSVCRT package instead of the newer one using UCRT (i.e., MINGW64 over UCRT64) for compatibility (no need to distribute UCRT on Windows < 10). See https://www.msys2.org/docs/environments/